### PR TITLE
Migrate Travis to GitHub Actions

### LINKED ISSUE #190: Create GIthub Actions to automatically build and test this package
We should create Github Actions to automatically build and test this package against various versions of PHP.

### PR DESCRIPTION
**Pull request by koriym**
Closes #190 #195 
Related #189

### CODE REVIEW

**Review by kenjis — APPROVED** [review]

**Comment by koriym** [comment]
@kenjis Someone else may have an opinion, so why don't you give it some time instead of merging it right away? (Very simple ones are of course not necessary.)

**Comment by kenjis** [comment]
@koriym Okay, I will wait more next time.

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: ${{ runner.os }}-composer-
+
+      - name: Install dependencies (5.6 and 7.x)
+        if: ${{ matrix.php-version != '8.0' }}
+        run: composer install --no-interaction --prefer-dist
+
+      - name: Install dependencies (8.0)
+        if: ${{ matrix.php-version == '8.0' }}
+        run: composer install --no-interaction --prefer-dist --ignore-platform-req=php
```
Comment (harikt) [inline .github/workflows/continuous-integration.yml:54]: Hi @koriym , 

I have removed this on https://github.com/auraphp/Aura.Sql/pull/209 . Not sure why you were testing only for 8.x separately. May be phpunit related ?

Comment (koriym) [reply .github/workflows/continuous-integration.yml:54]: Probably. No problem if it worked well without it!

**Pull request merged**